packages/cfd/src/types/cfd-store.types.ts

```diff
@@ -4,18 +4,34 @@ import { TCFDPasswordFormValues } from 'Containers/cfd-password-modal';
 import { TDerivezCompanies, TDxCompanies, TMtCompanies } from 'Stores/Modules/CFD/Helpers/cfd-config';
 import { FormikHelpers } from 'formik';
 
-type TStoreProofOfAddressArgs = {
+export type TStoreProofOfAddressArgs = {
     file_uploader_ref: HTMLDivElement | null;
     values: {
         [key: string]: string;
     };
 };
 
+export type TAccountType = {
+    category: 'real' | 'demo' | '';
+    type?:
+        | 'all'
+        | 'all_svg'
+        | 'derivez'
+        | 'dxtrade'
+        | 'synthetic'
+        | 'synthetic_svg'
+        | 'financial'
+        | 'financial_svg'
+        | 'financial_stp'
+        | ''
+        | string;
```

Comment:
@hamza-deriv to keep the autocompletion for types you can use below approach:
```suggestion
        | Omit<string, 'all'
        | 'all_svg'
        | 'derivez'
        | 'dxtrade'
        | 'synthetic'
        | 'synthetic_svg'
        | 'financial'
        | 'financial_svg'
        | 'financial_stp' >;
```